manifest_staging/charts/csi-secrets-store-provider-azure/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.pspEnabled }}
```

Comment:
```yaml
{{- if .Values.rbac.pspEnabled }}
apiVersion: policy/v1beta1
kind: PodSecurityPolicy
metadata:
  name: {{ template "sscdpa.psp.fullname" . }}
spec:
  hostNetwork: true
  seLinux:
    rule: RunAsAny
  privileged: {{ .Values.linux.privileged }}
  volumes:
    - hostPath
    - secret
  hostPorts:
    - min: 0
      max: 65535
  fsGroup:
    ranges:
    - max: 65535
      min: 1
    rule: MustRunAs
  runAsUser:
    rule: RunAsAny
  supplementalGroups:
    ranges:
    - max: 65535
      min: 1
    rule: MustRunAs
{{- end }}
```

Reply:
Also, please include the default labels in the `PodSecurityPolicy`, `Role` and `RoleBinding`

Reply:
@aramase should this match deployment PSP here? https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/deployment/pod-security-policy.yaml#L57-L73

Reply:
@sozercan Apart from the `allowedHostPaths `, the rest of the spec is very similar here. Imposing the different host paths can get tedious if users want to mount custom env file or more. WDYT? 

Also, that file seems to be a little outdated. We should update it in a follow up PR.

Reply:
sounds good. we can have a follow up to address that.

Reply:
Opened https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/479 to track the change.